From 25f13f3058c79e3f4a2b1cd257b213f102931cc3 Mon Sep 17 00:00:00 2001
From: Dave Flogeras <dflogeras2@gmail.com>
Date: Wed, 6 Jan 2021 20:07:20 -0400
Subject: [PATCH] Make test static library explicitly link against kColorPicker
 and X11.

This was broken when those libraries were made part of a PRIVATE
interface by bbc2b6595 and af2ff6167.
---
 tests/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index b1ecc8a..9918010 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -56,7 +56,7 @@ set(TESTUTILS_SRC
 
 add_library(KIMAGEANNOTATOR_STATIC STATIC ${KIMAGEANNOTATOR_SRCS})
 
-target_link_libraries(KIMAGEANNOTATOR_STATIC Qt5::Widgets Qt5::Svg kImageAnnotator)
+target_link_libraries(KIMAGEANNOTATOR_STATIC Qt5::Widgets Qt5::Svg kImageAnnotator kColorPicker::kColorPicker X11)
 
 target_compile_definitions(KIMAGEANNOTATOR_STATIC PRIVATE KIMAGEANNOTATOR_LANG_INSTALL_DIR="${KIMAGEANNOTATOR_LANG_INSTALL_DIR}")
 
